### PR TITLE
add task solution Ts_sort_students

### PR DESCRIPTION
**Pull request by TamaraShukalo**

### CODE REVIEW

**Review by YegorKorenkov — APPROVED** [review]
Good job!

File: src/sortStudents.ts [hunk 1]
```diff
@@ -1,16 +1,64 @@
...
+    let result: number = 0;
+
+    switch (sortBy) {
+      case SortType.Name:
+        result = student1.name.localeCompare(student2.name);
+        break;
+
+      case SortType.Surname:
+        result = student1.surname.localeCompare(student2.surname);
+        break;
```
Comment (And678) [inline src/sortStudents.ts:41]: Check the checklist. You can combine these. Also the same applies to Age and Married
```suggestion
      case SortType.Name:
      case SortType.Surname:
        result = student1[sortBy].localeCompare(student2[sortBy]);
        break;
```

**Review by And678 — CHANGES_REQUESTED** [review]
Gj, only one thing to improve here:

**Review by anstsot — APPROVED** [review]
Good work, approving:sparkles:

**Pull request closed without merging**